hls/xls/meta.yaml

```diff
@@ -0,0 +1,53 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'| format(GIT_DESCRIBE_TAG | default('0.0.0') , GIT_DESCRIBE_NUMBER | default(0) | int, GIT_DESCRIBE_HASH | default('gUNKNOWN')) %}
+
+package:
+  name: xls
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/google/xls.git
+    git_rev: main
+    patches:
+      - add-dist-rule.patch
+      - skip-python-requirements.patch
+
+build:
+  # number: 202202082028
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20220208_2028
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+
+requirements:
+  build:
+    - bazel
+  host:
+    - python
+    - pip
+    - pycparser
+    - Flask
+    - Jinja2
+    - werkzeug
+    - itsdangerous
+    - click
+    - protobuf
+    - numpy
+    - scipy
```

Comment:
Not sure why but this set of dependencies is a hell for Conda and rendering it for building takes unusually long time.
Shouldn't at least some of the versions match https://github.com/google/xls/blob/692e37534dfcd1af41b0b8b04ff2004d6a8193c5/dependency_support/pip_requirements.txt ?

I can see even setting a version for only `numpy` reduces the rendering time a lot so please set at least that one.

You can use `- numpy >=1.19.5,<=1.20` to have it the same as in the `requirements.txt` file or simply `- numpy 1.20` to have it use any `1.20.*` version.

Reply:
you're right we should definitly have tighter version if it helps conda converge quicker, I'll see if we can get rid of the dependencies in the first place because we're not distributing the corresponding binary https://github.com/google/xls/blob/bca00e53bdbe0e4428f7d05009395602796402ff/xls/visualization/ir_viz/BUILD#L43